core/flyout_vertical.js

```diff
@@ -149,13 +149,21 @@ Blockly.VerticalFlyout.prototype.position = function() {
   var edgeHeight = targetWorkspaceMetrics.viewHeight - 2 * this.CORNER_RADIUS;
   this.setBackgroundPath_(edgeWidth, edgeHeight);
 
-  var y = targetWorkspaceMetrics.absoluteTop;
-  var x = targetWorkspaceMetrics.absoluteLeft;
-  if (this.toolboxPosition_ == Blockly.TOOLBOX_AT_RIGHT) {
-    x += (targetWorkspaceMetrics.viewWidth - this.width_);
-    // Save the location of the left edge of the flyout, for use when Firefox
```

Comment:
@BeksOmega digging this up because I'm chasing why ``this.leftEdge_`` was removed here, but not [here](https://github.com/google/blockly/blob/master/core/flyout_vertical.js#L355). 

If you can remember, I'm wondering if you think this was deliberate or accidental?

Reply:
I can't remember, but I can't imagine it being deliberate.

Reply:
Okay, I'll look into whether this code is still needed.

Reply:
I couldn't reproduce #1425 on the latest firefox if that helps. And it looks like the root bug [1066435 ](https://bugzilla.mozilla.org/show_bug.cgi?id=1066435)was closed.

Reply:
Cool, good to know. Thanks